src/contributing/void-docs/style-guide.md

```diff
@@ -6,6 +6,19 @@ This style guide outlines the standards for contributing to the
 [mdBook](https://rust-lang-nursery.github.io/mdBook/) stored in the
 [void-docs](https://github.com/void-linux/void-docs/) repository.
 
+## General
+
+Step-by-step instructions containing 'magic' commands for copying-and-pasting
+are strongly discouraged. Users are expected to read the canonical documentation
+(e.g. man pages) for individual programs to understand how to use them, rather
+than relying on blindly copying-and-pasting.
```

Comment:
blindly might be too much? we want to keep this friendly :)

Reply:
Sure. :-) Removed.